docs/detections/rules-ui-create.asciidoc

```diff
@@ -29,13 +29,15 @@ alert is generated for every source IP address that appears in at least 10 of
 the rule's search results.
 * <<create-eql-rule, *Event correlation*>>: Searches the defined indices and creates an alert when results match an
 {ref}/eql.html[Event Query Language (EQL)] query.
-* <<create-indicator-rule, *Indicator match*>>: Creates an alert when {es-sec} index field values match field values defined in the specified indicator index patterns. For example, you can create an indicator index for IP addresses and use this index to create an alert whenever an event's `destination.ip` equals a value in the index. Indicator index field mappings should be {ecs-ref}[ECS-compliant]. For information on creating {es} indices and field types, see
+* <<create-indicator-rule, *Indicator match*>>: Creates an alert when {elastic-sec} index field values match field values defined in the specified indicator index patterns. For example, you can create an indicator index for IP addresses and use this index to create an alert whenever an event's `destination.ip` equals a value in the index. Indicator index field mappings should be {ecs-ref}[ECS-compliant]. For information on creating {es} indices and field types, see
 {ref}/getting-started-index.html[Index some documents],
 {ref}/indices-create-index.html[Create index API], and
-{ref}/mapping-types.html[Field data types]. If you have indicators in a standard file format, such as CSV or JSON, you can also use the Machine Learning Data Visualizer to import your indicators into an indicator index. See https://www.elastic.co/guide/en/machine-learning/7.10/ml-gs-visualizer.html and use the Import Data option to import your indicators.
+{ref}/mapping-types.html[Field data types]. If you have indicators in a standard file format, such as CSV or JSON, you can also use the Machine Learning Data Visualizer to import your indicators into an indicator index. See {ml-docs}/ml-getting-started.html#sample-data-visualizer[Explore the data in {kib}] and use the *Import Data* option to import your indicators.
 
 TIP: You can also use value lists as the indicator match index. See <<indicator-value-lists>> at the end of this topic for more information.
 
+* <<create-new-terms-rule, *New terms*>>: Generates an alert for each new term it detects in source documents.
```

Comment:
Can we add a bit to define "new term" here and how the History Window Size works? E.g. for a rule with an interval of 5 minutes, 0 additional lookback, and a history window size of 7 days, a term will be "new" if the only time it appears within the last 7 days is also within the last 5 minutes.

Reply:
+1 for a concise definition of "new term." Maybe for this first reference, generalize about time intervals and history windows, and provide those details in the "Create a new terms rule" section below? This sort of gets at that, though it could probably still use more explanation of what "new" means:
```suggestion
* <<create-new-terms-rule, *New terms*>>: Generates an alert for each new term detected in source documents within a specified time period.
```